var/spack/repos/builtin/packages/everybeam/package.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+# Contribute recipe
+from spack.package import *
+
+
+class Everybeam(CMakePackage):
+    """
+    The EveryBeam library is a library that provides the antenna response pattern for several instruments, 
+    such as LOFAR (and LOBES), SKA (OSKAR), MWA, JVLA, etc.
+    """
+
+    homepage = "https://wsclean.readthedocs.io/en/latest/"
+    git      = "https://git.astron.nl/RD/EveryBeam.git"
+    maintainers = ["pelahi"]
+
+    version('0.3.0', commit='2eea95e1d93832d73b623be85085f18875a14fa5')
+    version('0.2.0', commit='74fe444e0052d1179126ba4742eec8392336019d')
+
+    variant('python', default=False, description='Add python support')
+    variant('tests', default=False, description='Build tests')
```

Comment:
This shouldn't be a variant, if it doesn't change the nature of the binaries being produced.